### PR TITLE
Adjust `@configu/ts` - `ConfiguConfigStore` for the Revision Tag feature

### LINKED ISSUE #348: Adjust `ConfiguConfigStore` for the `Revision Tag` feature
### Suggestion

Change the `ConfiguConfigStore` to support the [`Revision Tag` feature](https://configu.com/docs/configs-history/#revision-tags) by allowing users to perform evaluations against tags they created in their organization.

### Motivation

Allows users of `ConfiguConfigStore` to always evaluate configs from a deterministic version of their config state without having to worry about unexpected changes.

### Tasks

- [x] TS SDK implementation
- [x] CLI implementation

### PR DESCRIPTION
**Pull request by RichardAkman**
Closes #348

### CODE REVIEW

File: ts/packages/cli/src/base.ts
Comment (RichardAkman) [inline ts/packages/cli/src/base.ts:1]: TODO: split into a follow up PR

File: ts/packages/cli/package.json
Comment (RichardAkman) [inline ts/packages/cli/package.json:1]: TODO: split into a follow up PR

**Review by RonConfigu — APPROVED** [review]
LGTM 💯

File: ts/packages/ts/src/stores/Configu.ts [hunk 1]
```diff
@@ -7,11 +7,18 @@ export type ConfiguConfigStoreConfiguration = {
   credentials: { org: string; token: string };
   endpoint?: string;
   source?: string;
+  tag?: string;
 };
 
 export class ConfiguConfigStore extends ConfigStore {
   private client: Axios;
-  constructor({ credentials, endpoint = `https://api.configu.com`, source = 'sdk' }: ConfiguConfigStoreConfiguration) {
+  private tag?: string;
```
Comment (rannn505) [inline ts/packages/ts/src/stores/Configu.ts:15]:
```suggestion
  public tag?: string;
```

**Review by rannn505 — APPROVED** [review]

**Pull request merged**